src/hotspot/share/jvmci/jvmciCompilerToVM.cpp

```diff
@@ -2349,13 +2349,25 @@ C2V_VMENTRY_PREFIX(jlong, getCurrentJavaThread, (JNIEnv* env, jobject c2vm))
   return (jlong) p2i(thread);
 C2V_END
 
-C2V_VMENTRY_PREFIX(jboolean, attachCurrentThread, (JNIEnv* env, jobject c2vm, jboolean as_daemon))
+C2V_VMENTRY_PREFIX(jboolean, attachCurrentThread, (JNIEnv* env, jobject c2vm, jbyteArray name, jboolean as_daemon))
   if (thread == NULL) {
     // Called from unattached JVMCI shared library thread
```

Comment:
What "unattached" means?

Reply:
This is a thread that was created and started in libgraal. Libgraal is a JNI library and so any thread it creates that wants to call into the VM must first [attach](https://docs.oracle.com/javase/7/docs/technotes/guides/jni/spec/invocation.html) itself to the VM.

Reply:
okay.